tests/test_scripts.py

```diff
@@ -211,7 +213,95 @@ def test_process_lane(self):
                 eq_(4, len(cached_feeds))
 
 
-
-
-
-
+class TestLoanReaperScript(DatabaseTest):
+
+    def test_reaping(self):
+
+        # This patron stopped using the circulation manager a long time
+        # ago.
+        inactive_patron = self._patron()
+
+        # This patron is still using the circulation manager.
+        current_patron = self._patron()
```

Comment:
We don't know that in general. For purposes of the test I'm setting up those two scenarios. If a patron has loans in the database that have expired, they clearly haven't synced since before those loans expired.